src/components/chapters/ChapterAndJuzList.tsx

```diff
@@ -3,27 +3,18 @@ import React, { useState, useMemo } from 'react';
 
 import classNames from 'classnames';
 import useTranslation from 'next-translate/useTranslation';
-import dynamic from 'next/dynamic';
 
 import CaretDownIcon from '../../../public/icons/caret-down.svg';
 import Link from '../dls/Link/Link';
-import Spinner from '../dls/Spinner/Spinner';
 import SurahPreviewRow from '../dls/SurahPreview/SurahPreviewRow';
 import Tabs from '../dls/Tabs/Tabs';
 
 import styles from './ChapterAndJuzList.module.scss';
+import JuzView from './JuzView';
```

Comment:
Is there a reason why `dynamic` was removed? we have added it to lazy load this component instead of it being part of the initial payload which makes it larger and slower to parse